vpr/src/route/route_tree_timing.h

```diff
@@ -19,7 +19,7 @@ void free_route_tree(t_rt_node* rt_node);
 void print_route_tree(const t_rt_node* rt_node);
 void print_route_tree(const t_rt_node* rt_node, int depth);
 
-t_rt_node* update_route_tree(t_heap* hptr, SpatialRouteTreeLookup* spatial_rt_lookup);
+t_rt_node* update_route_tree(t_heap* hptr, int target_pin, SpatialRouteTreeLookup* spatial_rt_lookup);
```

Comment:
Rename target_pin to target_net_pin_index, and if you know what the routine does, write a comment above the function name.

Reply:
I never added a comment because the description for these functions are in the .cpp. But I've just added to the comment to describe what the target_net_pin_index parameter means.

Reply:
Your call; you can keep following the .cpp convention for these files if you like.